spec/spec_helper.rb

```diff
@@ -1,7 +1,16 @@
-# Copied from https://github.com/cucumber/aruba/blob/3b1a6cea6e3ba55370c3396eef0a955aeb40f287/spec/spec_helper.rb
-# Licensed under MIT - https://github.com/cucumber/aruba/blob/3b1a6cea6e3ba55370c3396eef0a955aeb40f287/LICENSE
```

Comment:
why was this license note removed? 
the license text clearly expresses, that the license information MUST NOT removed, right?

Reply:
A license header like this is not applicable to a file name, but to the contents of the file. It is very clear that the code was copied from a specific place (Aruba) and inherited the license as a result of that.

Removing a license header like this is simply a matter of deleting all code in the file and writing it from scratch, which I did, and which is needed, because this project's spec helper will have nothing in common with Aruba's spec_helper.

Thus the contents of the file revert to being under the same license as the core project.

I'd be happy to add back a license header, but I do note that license headers are not universally applied in the project. Of course I'd need to add it back without the reference to aruba, since it no longer has any relevance.